src/activities/photo_hunter/Photo_hunter.qml

```diff
@@ -0,0 +1,172 @@
+/* GCompris - photo_hunter.qml
```

Comment:
Naming conventions implies Qml file to by CamelCase so this one must be PhotoHunter.qml